sdk/core/azure-core/azure/core/utils/_utils.py

```diff
@@ -99,6 +99,8 @@ def _case_insensitive_dict(*args, **kwargs):
         # multidict is installed by aiohttp
         from multidict import CIMultiDict
 
+        if len(kwargs) == 0 and len(args) == 1 and (not args[0]):
```

Comment:
What's `args[0]` in this case? Maybe add a small comment as to what this if-statement is addressing?

Reply:
This is to handle case_insensitive_dict(None). requests.structures.CaseInsensitiveDict returns empty dict while multidict.CIMultiDict raise error.

Here we want to return dict().